src/components/AreaContainer/AreaContainer.js

```diff
@@ -17,13 +19,32 @@ const AreaContainer = ({areas}) => {
         region_code={area.region_code}
         quick_search={area.quick_search}
         listings={area.listings}
+        displayListings={displayListings}
+        currentListings={currentListings}
       />
     )
   })
 
   return (
     <section className="areas-container">
       {areasToDisplay}
+      {/*<Route
```

Comment:
Had moved the route to area container, but ran into some issues and remembered that Robbie said the routes should live in App, so moved it to App and all current routes are working.